requirements.txt

```diff
@@ -0,0 +1,53 @@
+annotated-types==0.7.0
+anyio==4.6.0
+bcrypt==4.2.0
+certifi==2024.8.30
+cffi==1.17.1
+click==8.1.7
+colorama==0.4.6
+cryptography==43.0.1
+dnspython==2.7.0
+ecdsa==0.19.0
+email_validator==2.2.0
+fastapi==0.115.0
+fastapi-cli==0.0.5
+greenlet==3.1.1
+h11==0.14.0
+httpcore==1.0.6
+httptools==0.6.1
+httpx==0.27.2
+idna==3.10
+itsdangerous==2.2.0
+Jinja2==3.1.4
+markdown-it-py==3.0.0
+MarkupSafe==2.1.5
+mdurl==0.1.2
+orjson==3.10.7
+passlib==1.7.4
+psycopg2==2.9.9
+psycopg2-binary==2.9.9
+pyasn1==0.6.1
+pycparser==2.22
+pydantic==2.9.2
+pydantic-extra-types==2.9.0
+pydantic-settings==2.5.2
+pydantic_core==2.23.4
+Pygments==2.18.0
+python-dotenv==1.0.1
+python-jose==3.3.0
+python-multipart==0.0.12
+PyYAML==6.0.2
+redis==5.1.1
+rich==13.9.2
+rsa==4.9
+shellingham==1.5.4
+six==1.16.0
+sniffio==1.3.1
+SQLAlchemy==2.0.35
+starlette==0.38.6
+typer==0.12.5
+typing_extensions==4.12.2
+ujson==5.10.0
+uvicorn==0.31.0
+watchfiles==0.24.0
+websockets==13.1
```

Comment:
Explain what are websockets and why are they here, I have no idea why you have them here